docs/design/architecture.md

```diff
@@ -170,47 +397,89 @@ The following diagram illustrates the Kata Containers network hotplug workflow.
 ![Network Hotplug](arch-images/kata-containers-network-hotplug.png)
 
 ## Storage
-Container workloads are shared with the virtualized environment through [virtio-fs](https://virtio-fs.gitlab.io/).
 
-The [devicemapper `snapshotter`](https://github.com/containerd/containerd/tree/master/snapshots/devmapper) is a special case. The `snapshotter` uses dedicated block devices rather than formatted filesystems, and operates at the block level rather than the file level. This knowledge is used to directly use the underlying block device instead of the overlay file system for the container root file system. The block device maps to the top read-write layer for the overlay. This approach gives much better I/O performance compared to using `virtio-fs` to share the container file system.
+### virtio SCSI
+
+If a block-based graph driver is configured,
+`virtio-scsi` is used to "share" the workload image (such as
+`busybox:latest`) into the container's environment inside the VM.
+
+### virtio FS
+
+In all other cases a [`virtio-fs`](https://virtio-fs.gitlab.io)
+(`VIRTIO`) mount point is used instead. The [agent](#agent) uses this
+mount point as the root filesystem for the container processes.
+
+For `virtio-fs`, the [runtime](#runtime) starts one `virtiofsd` daemon
+(that runs in the host context) for each VM created.
+
```

Comment:
What about 9pfs? Is it no longer supported at all?  If it is still supported I would mention it. If it isn't supported I would add a note it has been deprecated since version (not sure when it happened) of Kata.

Reply:
Also, what about pmem?  How is that case covered?  I believe it works differently and uses map_sync.  That would be worth explaining.

Reply:
Sure lets defer to [3246](https://github.com/kata-containers/kata-containers/issues/3246)